test/unit/search_api_test.rb

```diff
@@ -26,7 +26,7 @@ class SearchAPITest < ActiveSupport::TestCase
 
   context "given a search scoped to a manual" do
     setup do
-      @manual_link = '/manual/manual-name'
+      @manual_link = 'manual/manual-name'
```

Comment:
Ah. I just did a blanket revert on the whole original commit. Amended to not change this line...